odoo/addons/base/data/res_country_data.xml

```diff
@@ -1485,7 +1485,7 @@
             <field eval="'%(street)s\n%(street2)s\n%(city)s %(state_code)s %(zip)s\n%(country_name)s'" name="address_format" />
             <field name="currency_id" ref="USD" />
             <field eval="1" name="phone_code" />
-            <field name="vat_label">EIN</field>
+            <field name="vat_label"/> <!-- Empty to reset on existing DBs-->
```

Comment:
The file is in noupdate="1", so I don't think it will reset for existing databases. It will only impact new ones.

Reply:
Ah, didn't see that. So I better remove the line then?

Reply:
You could leave it like this, but the comment isn't completely correct.

If we really want to empty the field for existing db, I suppose it could be done in master with a migration script.
cc @chklop

Reply:
Yep it's fine !

Reply:
Also, It is very discouraged to update `base` because it will update all the module, which can have many unexpected side effects

Reply:
@william-andre Problem is, with the changes made recently about the label of that field (only ever using the label of the res_company country), we always put "EID" as Tax ID for the US. But as said in related tax (3162675), it's the wrong label, as Tax IDs in the US are not always EIN.
What's the best way to proceed to have "Tax ID" for US then instead ?

Reply:
> only ever using the label of the res_company country

Not doing this as it is not correct in many cases

Reply:
I can revive this if you want to, it should solve most of your issues...
https://github.com/odoo/odoo/pull/79659

Reply:
> Not doing this as it is not correct in many cases

Still was decided like this by FP last month: https://github.com/odoo/odoo/commit/7085a5b0c4ed2509d16dde7c1f50c524bef912a2

Reply:
@chklop Okay, so what do we do with this?

Reply:
@dylankiss @william-andre I don't know, what's the deal ? Basically, Tax ID should not be renamed to EIN in the US. If it's a bad idea to correct it on older DBs for some reason (that I did not really understand), then let's at least correct it on new ones (master).